src/lib/crypt_ops/crypto_rand_numeric.c

```diff
@@ -154,6 +154,28 @@ crypto_fast_rng_get_uint64(crypto_fast_rng_t *rng, uint64_t limit)
                   crypto_fast_rng_getbytes(rng, (void*)&val, sizeof(val)));
 }
 
+/**
+ * As crypto_rand_u32, but extract the result from a crypto_fast_rng_t.
+ */
+uint32_t
+crypto_fast_rng_get_u32(crypto_fast_rng_t *rng)
+{
+  IMPLEMENT_RAND_UNSIGNED(uint32_t, UINT32_MAX, UINT32_MAX,
```

Comment:
This won't actually work so well -- the "limit" argument has to be one more than the largest possible output. 
Instead, just do something like
    uint32_t val;
    crypto_fast_rng_getbytes(rng, (void*)&val, sizeof(val)));
    return val;

Reply:
Pushed fixup as c83fedb284.